<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BirthdayListTableConfig</class>
 <widget class="QWidget" name="BirthdayListTableConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lblTitleColumns">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Columns</string>
     </property>
    </widget>
   </item>
   <item row="17" column="0">
    <spacer name="spacerScreenBottom">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblShowColumnHeaders">
     <property name="text">
      <string>Show column headers:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="chckShowColumnHeaders">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <spacer name="spacerColumnsBottom">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="chckShowColDate">
     <property name="text">
      <string>Date</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblShowColumns">
     <property name="text">
      <string>Show columns:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QCheckBox" name="chckShowColWhen">
     <property name="text">
      <string>When</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="chckShowColName">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QCheckBox" name="chckShowColAge">
     <property name="text">
      <string>Age</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="lblTitleNames">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Names</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="lblShowNicknames">
     <property name="text">
      <string>Show nicknames if available:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <spacer name="spacerNamesBottom">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="13" column="1">
    <widget class="QCheckBox" name="chckShowNicknames">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <spacer name="spacerScreenRight">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="lblTitleDateFormat">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Date format</string>
     </property>
    </widget>
   </item>
   <item row="16" column="0">
    <widget class="QLabel" name="lblDateDisplayFormat">
     <property name="text">
      <string>Date display format:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="16" column="1" colspan="2">
    <widget class="QComboBox" name="cmbDateDisplayFormat">
     <item>
      <property name="text">
       <string notr="true">31. 1.</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">31. 01.</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">31-1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">31-01</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">1-31</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">01-31</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">31/1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">31/01</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">1/31</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">01/31</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>chckShowColumnHeaders</tabstop>
  <tabstop>chckShowColName</tabstop>
  <tabstop>chckShowColAge</tabstop>
  <tabstop>chckShowColDate</tabstop>
  <tabstop>chckShowColWhen</tabstop>
  <tabstop>chckShowNicknames</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
